README.md

```diff
@@ -10,6 +10,40 @@ This repository hosts scripts to generate self-contained installations of the [E
 
 ------
 
+
+### 1.1. Prerequisites
+
+If you don't have Git, install it. You can install Git from websites such as [Git Downloads](https://git-scm.com/downloads).
+
+The bundled installer requires the following prerequisites for each operating system.
```

Comment:
Python, and not the bundled installer, requires them.

Reply:
Will update.